rules/sha/index.bzl

```diff
@@ -0,0 +1,10 @@
+# Calculates a sha of the contents of the files in `srcs` and outputs it to a file called `name`.txt.
+def sha(name, srcs, **kwargs):
```

Comment:
nit: consider calling this sha1 (or add to the documentation above?)

Reply:
Added to the documentation - would add an arg if folks wanted to change which sha algo to use.